### PR TITLE
users/daniellemott/change variable pipeline

### PR DESCRIPTION
**Pull request by daniellemott**
This PR fixes two mistakes that were made in https://github.com/microsoft/adaptivecards-templates/pull/86#pullrequestreview-363777009. 

In router.test.ts, secrets.AZURE_ADAPTIVECMS_CLIENT_SECRET is changed to CLIENT_SECRET_TOKEN. In app.ts, secrets.DB_CONNECTION_TOKEN is changed to DB_CONNECTION_TOKEN. It also fixes the formatting of router.test.ts and removes unnecessary string token replacement in the dev/stage pipelines.

### CODE REVIEW

File: .github/workflows/azure-dev.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
                 files: '["**/*.ts"]'
               env:
                 CLIENT_SECRET_TOKEN: ${{ secrets.AZURE_ADAPTIVECMS_CLIENT_SECRET }}
+                DB_CONNECTION_TOKEN: ${{ secrets.AZURE_ADPATIVECMS_DB_CONNECTION_STRING }}
```
Comment (rachel-slater) [inline .github/workflows/azure-dev.yml]: Is this typo ok? 'Adpative'

File: .github/workflows/azure-dev.yml [hunk 2]
```diff
@@ -57,12 +57,6 @@ jobs:
                   cd ..
                   npm run test
                   cd ..
-            - name: Find and Replace DB_CONNECTION_TOKEN
-              uses: cschleiden/replace-tokens@v1
```
Comment (rachel-slater) [inline .github/workflows/azure-dev.yml]: I'm not sure about using another open-source repo (with only 3 stars). Maybe ask Mark if this is okay or not?
I looked at the src files & it's not alot of code, so you could replicate it in your own repo.

File: .github/workflows/azure-stage.yml [hunk 3]
```diff
@@ -36,6 +36,7 @@ jobs:
                 files: '["**/*.ts"]'
               env:
                 CLIENT_SECRET_TOKEN: ${{ secrets.AZURE_ADAPTIVECMS_CLIENT_SECRET }}
+                DB_CONNECTION_TOKEN: ${{ secrets.AZURE_ADPATIVECMS_DB_CONNECTION_STRING }}
```
Comment (rachel-slater) [inline .github/workflows/azure-stage.yml]: typo again.

File: .github/workflows/azure-dev.yml (hunk 1 above)
Comment (daniellemott) [reply .github/workflows/azure-dev.yml]: Oops I misspelled the actual Github secret. Fixed.

File: .github/workflows/azure-dev.yml (hunk 2 above)
Comment (daniellemott) [reply .github/workflows/azure-dev.yml]: I talked to Mark and Stephane. Stephane is registering our use of this open source repo. He said it's okay for us to use it in our pipeline.

Comment (rachel-slater) [reply .github/workflows/azure-dev.yml]: Okay, sounds good!

**Pull request merged**